File: PHP/1005  - Media 1.php
<?php

$nota_1 = (double)readline();
$nota_2 = (double)readline();

$media = ($nota_1*3.5 + $nota_2*7.5) / 11;

$media = number_format($media, 5, ".", "");

echo "MEDIA = $media\n"

?>

File: Java/1005 - Media 1.java
import java.util.Scanner;

class Main {
    public static void main(String[] args){
        
        Scanner input = new Scanner(System.in);

        double nota_1 = input.nextDouble();
        double nota_2 = input.nextDouble();

        double media = (nota_1*3.5 + nota_2*7.5) / 11;

        System.out.printf("MEDIA = %.5f\n", media);
    }
}